filebrowser/base.py

```diff
@@ -219,13 +219,13 @@ def __init__(self, path, site=None):
         else:
             self.path = path
         self.head = os.path.dirname(path)
-        self.filename = os.path.basename(path)
+        self.filename = force_text(os.path.basename(path))
```

Comment:
I'm not sure about this one: With my setup, both self.filename and force_text(self.filename) is unicode (I'm using python 2.7). I'm not exactly sure about the consequences of changing this line ... therefore, I tend to leave it as is. Theoretically speaking, force_text seems to make sense. But I don't wanna run into backwards-incompatible updates here.